shared/controller-api/src/main/swagger/Controller.yaml

```diff
@@ -284,8 +284,53 @@ paths:
           description: Scope or stream with given name not found
         500:
           description: Internal server error while updating the stream state
-
+  /scopes/{scopeName}/streams/{streamName}/scaling-events:
+    parameters:
+      - in: path
+        name: scopeName
+        description: Scope name
+        required: true
+        type: string
+      - in: path
+        name: streamName
+        description: Stream name
+        required: true
+        type: string
+    get:
+      parameters:
+        - in: query
+          name: from
+          description: Parameter to display scaling events from that particular datetime. Input should be milliseconds from Jan 1 1970.
+          required: true
+          type: integer
+          format: int64
+        - in: query
+          name: to
+          description: Parameter to display scaling events to that particular datetime. Input should be milliseconds from Jan 1 1970.
+          required: true
+          type: integer
+          format: int64
+      operationId: getScalingEvents
+      description: Get scaling events for a given datetime period.
+      produces:
+        - application/json
+      responses:
+        200:
+          description: List of scaling events
+          schema:
+            $ref: "#/definitions/ScalingEventList"
+        404:
+          description: Scope/Stream not found
+        500:
+          description: Server error
```

Comment:
Expand message to be more descriptive since we will be generating public document from this (see other description after merging with master).